Add two TF modules for brig (#244)
wireapp/wire-server-deploy: terraform/modules/aws-brig-email-sending/README.md

Terraform module: Brig to enable email sending
==============================================

> This module creates AWS resources (SES, SQS, SNS) enabling Brig to send emails.

Reviewer: ```suggestion
Wire-server's "brig" components needs to send emails. This can either be done by configuring an SMTP server (Option 1), or by using AWS resources (SES, SQS, SNS) (Option 2).

This terraform module enables brig to send emails using option 2.
```

> 

Reviewer: This module stores sensitive secrets inside terraform state file. Please add a comment in the readme warning people about this (i.e. ensure to encrypt the terraform state).

This module is also untested, and while we can merge an initial version to test the functionality and improve on subsequent PRs, please indicate that this code is not ready yet in the readme. See also the readmes of the other terraform modules for this.

#### How to use the module

```hcl-terraform
module "bring_email_sending" {
  source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws-brig-email-sending?ref=develop"
  
  environment = "staging"

>   sender_address = "no-reploy@wire.com"

Reviewer: ```suggestion
  sender_address = "no-reply@example.com"
```

Terraform module: Brig to enable email sending
==============================================

State: __experimental__

> Wire-server's "brig" components needs to send emails. This can either be done by configuring an

Reviewer 2: ```suggestion
Wire-server's "brig" components needs to send emails and scheduling user deletions. This can either be done by configuring an
```

Reviewer 2: Nevermind my comment

SMTP server (Option 1), or by using AWS resources (Option 2).

This terraform module enables brig to send emails using option 2. In addition, it configures *MAIL FROM* for outgoing
emails, but does not enable incoming emails (possible solution: `aws_ses_receipt_rule`).

AWS resources: SES, SQS, SNS, DNS

#### Important note

This module causes Terraform to store sensitive data in the `.tfstate` file. Hence, encrypting the state should be
mandatory.

#### How to use the module

```hcl
module "brig_email_sending" {
  source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws-brig-email-sending?ref=develop"
  

>   environment = "staging"

Reviewer 2: ```suggestion
  environment = "dev"
```

Really nit-picky 🙈

Author: it defaults to `dev` actually, that is why I decided to give it a different value, otherwise it could be omitted altogether. But then, the example doesnt really makes sense anymore.
